Address Eliot's comments on #240 (#248)
IETF-OPSAWG-WG/draft-opsarea-rfc5706bis: draft-ietf-opsawg-rfc5706bis.md

   how threats should be mitigated, and then suggests appropriate standard
   protocols that could mitigate the threats.

>    It is not the intention that a protocol specification document should be held up waiting for operations and management solutions, or for supporting tooling, to be developed.  This is particularly the case when a protocol extension is proposed, but the base protocol is missing operations or management solutions.  However, it is the intent that new documents should clearly articulate the operations and management of that new work to fill any such gaps. Some operational and tooling guidance can only be determined once operational experience has been gained.  In such cases authors should document what is reasonably foreseeable at design time rather than prematurely specifying detailed mechanisms that may later prove to be wrong or unnecessarily constraining.

Reviewer: ```suggestion
   It is not the intention that a protocol specification document should be held up waiting for operations and management solutions, or for supporting tooling, to be developed.  This is particularly the case when a protocol extension is proposed, but the base protocol is missing operations or management solutions.  However, it is the intent that new documents should clearly articulate the operations and management of that new work to fill any such gaps. Some operational and tooling guidance can only be determined through deployment experience. In such cases, authors should document what is reasonably foreseeable at design time.
```

Reviewer 2: I don't know what "leave room the rest" mean?

Author: Yeah.  I wanted to discuss this with the rest of the group and not just merge it.  Selfishly, I like my original text better.

Reviewer 2: agreed Joe on the preference for your original text
